conda/recipes/pylibraft/meta.yaml

```diff
@@ -42,15 +52,20 @@ requirements:
     - scikit-build >=0.13.1
     - setuptools
   run:
-    - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
-    - cuda-python >=11.7.1,<12.0
+    {% if cuda_major == "11" %}
+    - cuda-python >=11.7.1,<12.0a0
```

Comment:
You can remove `cuda-python` here, it has a `run_export` that brings in a compatible version listed from `host`. https://github.com/conda-forge/cuda-python-feedstock/blob/8f1cd6b6123e45dd180957f4bd4a7c711ca118c0/recipe/meta.yaml#L32
```suggestion
```